dev/breeze/src/airflow_breeze/commands/developer_commands.py

```diff
@@ -129,6 +130,18 @@
 CELERY_INTEGRATION = "celery"
 
 
+def is_wsl() -> bool:
+    """Detect if we are running inside WSL."""
+    if platform.system().lower() != "Linux":
```

Comment:
It seems to be `linux` instead of `Linux` ?
```suggestion
    if platform.system().lower() != "linux":
```